src/old/modules/experts/components/ExpertInfo.tsx

```diff
@@ -36,7 +36,7 @@ const ExpertInfo: React.FC<IExpertInfoProps> = ({ expert }) => {
           </Box>
         </Box>
         <Grid container item className={classes.accordionWrapper}>
-          {expert.email || expert.socialNetwork ? (
+          {expert.email || expert.socialNetworks.length > 0 ? (
```

Comment:
nit: better keep consistency with the same check: either both do `.length > 0` or both do `.length !== 0`. Given that `length` cannot be negative, I'm rather for second version

Reply:
done